recipes/pybind_json/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pybind11_json" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "1904b1da077261bf156a937c6131db96b64d13aa0e09719f6bdf2bde8ed46d67" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pybind/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - nmake  # [win]
+    - make  # [unix]
+  host:
+    - pybind11 >=2.2.4,<3.0
+    - nlohmann_json >=3.2.0,<4.0
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/pybind11_json                                                                # [unix]
+    - test -f ${PREFIX}/include/pybind11_json/pybind11_json.hpp                                              # [unix]
```

Comment:
Does this project generate just headers? or does it also generate shared/static objects?

Reply:
Only one header